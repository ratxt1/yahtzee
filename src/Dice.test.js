import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Dice from "./Dice";


it("renders without crashing", function() {
  shallow(<Dice dice={[1,2]} locked={[true, false]} toggleLocked={() => {}} />);
});

it("matches snapshot", function() {
  let wrapper = shallow(<Dice dice={[1,2]} locked={[true, false]} toggleLocked={() => {}} />);
  let serialized = toJson(wrapper);
  expect(serialized).toMatchSnapshot();
});