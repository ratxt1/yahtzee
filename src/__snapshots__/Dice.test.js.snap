// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="Dice"
>
  <Die
    idx={0}
    key="0"
    locked={true}
    toggleLocked={[Function]}
    val={1}
  />
  <Die
    idx={1}
    key="1"
    locked={false}
    toggleLocked={[Function]}
    val={2}
  />
</div>
`;
